internal/logger/audit.go

```diff
@@ -86,10 +88,10 @@ func (lrw *ResponseWriter) Write(p []byte) (int, error) {
 // Write the headers into the given buffer
 func (lrw *ResponseWriter) writeHeaders(w io.Writer, statusCode int, headers http.Header) {
 	n, _ := fmt.Fprintf(w, "%d %s\n", statusCode, http.StatusText(statusCode))
-	lrw.bytesWritten += n
+	lrw.headerBytesWritten += n
```

Comment:
maybe we still need to add 'n' to lrw.bytesWritten here. This is not a 'written request body' but 'bytes written'

Reply:
@vadmeste I noticed that this metric included response headers size in addition to response body. If that's the intention, then we don't need this PR.

Reply:
We need them to separate otherwise we may be providing false information.

Reply:
> We need them to separate otherwise we may be providing false information.

I think maybe let's avoid counting headers in the response to make it consistent with InputBytes and we do not have to publish txHeaders for now.

Reply:
@vadmeste no we need to keep it to have a good understanding of how much is tx vs txHeaders for example in one of the customers the metadata was HTTP headers >> object size

Adding this is harmless..